docs/example_notebooks/example_echopop_workflow.ipynb

```diff
@@ -0,0 +1,2325 @@
+{
```

Comment:
<p>This is an interesting solution that you have!</p><p> </p><p>You can achieve the same effect via admonitions supported by MyST.</p><p><a href="https://jupyterbook.org/en/stable/content/content-blocks.html#notes-warnings-and-other-admonitions" target="_blank">https://jupyterbook.org/en/stable/content/content-blocks.html#notes-warnings-and-other-admonitions</a></p><p> </p><p>Same with the TIP block.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OSOceanAcoustics/echopop/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>Oh, that is much easier than what I was implementing! </p>